src/Slider.jsx

```diff
@@ -35,110 +39,121 @@ class Button extends React.Component {
   }
 }
 
+/* istanbul ignore next */
 const propTypes = {
+  ...withStylesPropTypes,
+
+  // Automatically adds a top position for large when enabled
+  autoAdjustVerticalPosition: PropTypes.bool,
+
   // the algorithm to use
   algorithm: PropTypes.shape({
     getValue: PropTypes.func,
     getPosition: PropTypes.func,
   }),
+
   // any children you pass in
-  children: PropTypes.node,
-  // standard class name you'd like to apply to the root element
+  children: PropTypes.any,
+
+  // class name prop
   className: PropTypes.string,
+
   // prevent the slider from moving when clicked
   disabled: PropTypes.bool,
+
   // a custom handle you can pass in
   handle: PropTypeReactComponent,
+
   // the maximum possible value
   max: PropTypes.number,
+
   // the minimum possible value
   min: PropTypes.number,
+
+  // step value
+  step: PropTypes.number,
+
   // called on click
   onClick: PropTypes.func,
+
   // called whenever the user is done changing values on the slider
   onChange: PropTypes.func,
+
   // called on key press
   onKeyPress: PropTypes.func,
+
   // called when you finish dragging a handle
   onSliderDragEnd: PropTypes.func,
+
   // called every time the slider is dragged and the value changes
   onSliderDragMove: PropTypes.func,
+
   // called when you start dragging a handle
   onSliderDragStart: PropTypes.func,
+
   // called whenever the user is actively changing the values on the slider
   // (dragging, clicked, keypress)
   onValuesUpdated: PropTypes.func,
+
   // the orientation
-  orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+  orientation: OrientationPropType,
+
   // a component for rendering the pits
   pitComponent: PropTypeReactComponent,
+
   // the points that pits are rendered on
   pitPoints: PropTypeArrOfNumber,
+
   // a custom progress bar you can pass in
   progressBar: PropTypeReactComponent,
-  // should we snap?
-  snap: PropTypes.bool,
-  // the points we should snap to
-  snapPoints: PropTypeArrOfNumber,
-  // whether a proposed update is valid
-  getNextHandlePosition: PropTypes.func,
+
   // the values
   values: PropTypeArrOfNumber,
 };
 
+/* istanbul ignore next */
```

Comment:
this ignore isn't necessary on the propTypes or the defaultProps (they will technically be run in tests).